assessments/Dibels_8_Benchmark/templates/studentAssessments.jsont

```diff
@@ -119,21 +119,21 @@
           "resultDatatypeTypeDescriptor": "uri://ed-fi.org/ResultDatatypeTypeDescriptor#Level",
           "result": "{{obj[2][1]}}"
         }
-        {% if obj[3] is defined and obj[3][1] != '' %}
-        ,
+          {% if obj[3] is defined and obj[3][1] != '' %}
+          ,
+          {
+            "assessmentReportingMethodDescriptor": "uri://dibels.uoregon.edu/assessment/dibels/AssessmentReportingMethodDescriptor#{{obj[3][0]}}",
+            "resultDatatypeTypeDescriptor": "uri://ed-fi.org/ResultDatatypeTypeDescriptor#Level",
+            "result": "{{obj[3][1]}}"
+          }
+          {% endif %}
+        {% elif obj[3] is defined and obj[3][1] != '' %}
```

Comment:
@susanxiong is this if statement here exactly the same as the one above?

Reply:
yes but the one above is nested under another if statement. you have to expand to see it though

Reply:
Oh yeah I'm following this now, looks like a method for handling trailing/leading commas!